packages/shopify-codemod/transforms/convert-default-export-objects-to-named-exports.js

```diff
@@ -0,0 +1,21 @@
+export default function convertDefaultExportObjectsToNamedExports({source}, {jscodeshift: j}, {printOptions = {quote: 'single'}}) {
+
+  return j(source)
+    .find(j.ExportDefaultDeclaration)
+    .forEach((nodePath) => {
+      const exportIndex = nodePath.parentPath.node.body.findIndex((prop) => prop.type === 'ExportDefaultDeclaration');
+      const body = nodePath.parentPath.value;
+      delete body[exportIndex];
+      const declarations = nodePath.node.declaration.properties;
+
+      let addedIndex = 0;
+      for (const dec of declarations) {
+        const varDec = j.variableDeclarator(dec.key, dec.value);
```

Comment:
The const per AST element was good for walking through the problem, but I think it's okay to compress it down to this now:
```js
        const namedExport = j.exportNamedDeclaration(
          j.variableDeclaration('const', [
            j.variableDeclarator(dec.key, dec.value)
          ])
        );
```